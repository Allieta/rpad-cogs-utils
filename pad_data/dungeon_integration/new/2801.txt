Daily Py Dungeon [17] - One-Shot Flampy - 1

stage 0

1547 - Flampy @ level 1
Basic attacks (1)

