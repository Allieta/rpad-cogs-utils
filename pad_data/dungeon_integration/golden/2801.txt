Daily Py Dungeon [17] - One-Shot Flampy

stage 0

1547 - Flampy @ level 1

This monster will only use basic attacks (or computing enemy skills failed)
